packages/sheets-ui/src/controllers/shortcuts/value.shortcut.ts

```diff
@@ -27,3 +27,10 @@ export const ClearSelectionValueShortcutItem: IShortcutItem = {
     binding: KeyCode.DELETE,
     mac: KeyCode.BACKSPACE,
 };
+
+export const ClearSelectionValueBackspaceShortcutItem: IShortcutItem = {
+    id: ClearSelectionContentCommand.id,
+    // when focusing on any other input tag do not trigger this shortcut
+    preconditions: (contextService) => whenSheetEditorFocused(contextService),
+    binding: KeyCode.BACKSPACE,
```

Comment:
In Excel, this behavior should be "start editing with an empty cell editor". The `ClearSelectionContentCommand` is carried by `KeyCode.DELETE`, and already implemented in the lines above.